github/packages/github/github.js

```diff
@@ -12,10 +12,160 @@ const axios = require('axios');
 async function _command(params, commandText, secrets = {}) {
 
   return {
-    response_type: 'in_channel', // or `ephemeral` for private response
-    text: await axios.get('https://raw.githubusercontent.com/nimbella/command-sets/master/github/README.md')
-    .then(response => { return response.data; })
-    .catch(error => { return error.response.data; })
+    "blocks": [
+      {
+        "type": "section",
+        "text": {
+          "type": "mrkdwn",
+          "text": "Nimbella Commander GitHub Command Set.\
+          \n_A Nimbella Commander command set for interfacing with GitHub._\
+          \n\n*Features*:\
+          \n• Create an issue\
+          \n• Close an issue\
+          \n• Reopen an issue\
+          \n• Find pull requests by date\
+          \n• Label an issue/pull request\
+          \n• Request someone to review a pull request\
+          \n• View repository community statistics\
+          \n• View recent pull requests\
+          \n• View GitHub command set documentation\
+          \n\n*Install*:\
+          \n`/nc csm_install github`\
+          \n\n*Requirements*:\
+          \nYou need to set up a Personal Access Token on GitHub with repo access. And save the token as a secret with the key `github_token`\
+          \ntype `/nc secret_create` to create secrets.\
+          \n<https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line|Click> to learn how to setup a Personal Access Token on GitHub\
+          \n\n*Usage*:"
+        }
+      },
+      // 
+      {
+        "type": "section",
+        "text": {
+          "type": "mrkdwn",
+          "text": "*Creating an issue.*. Type `/nc github_create_issue <repo> <title> <body>`"

```

Comment:
```suggestion
          "text": "*Creating an issue.* Type `/nc github_create_issue <repo> <title> <body>`"

```
Remove the extra dot. 

And what do you think of having it like this instead:
```js
text: "*Creating an issue:*\n`/nc github_create_issue <repo> <title> <body>`"
```
Output: 
![creating_issue_github_help_slack](https://user-images.githubusercontent.com/29819102/79173791-3dd57100-7e16-11ea-9332-5edaa91dff51.png)

Reply:
sure, thanks, we can improve it, I pulled the text from the existing readme as it was, but it can be made shorter.